api/urls.py

```
@@ -50,6 +50,7 @@
     path("create-thread/", ratelimit(key='ip', method='GET', rate='10/s')(create_thread_from_discord), name="create_thread_from_discord"),
 
     path("trips/", ratelimit(key='ip', method='GET', rate='10/s')(TripListView.as_view()), name="trip-list"),
+    path("trips/update_positions/", ratelimit(key='ip', method='GET', rate='2/m')(simulate_positions_view.as_view()), name="update-trip-positions"),
```

Ruff (0.14.8)

`simulate_positions_view` may be undefined, or defined from star imports

(F405)